pkgs/by-name/hy/hyprland-workspaces-tui/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  hyprland-workspaces,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-workspaces-tui-${version}";
+  version = "1.0.2";
+  src = fetchFromGitHub {
+    owner = "Levizor";
+    repo = "hyprland-workspaces-tui";
+    rev = "9eff75a4431193d1ebeb2e23bdefcd2dbcaa7666";
```

```suggestion
    tag = version;
```
Please tag your upstream releases